realm/realm-library/src/main/cpp/io_realm_internal_Util.cpp

```diff
@@ -22,7 +22,10 @@
 #include "mem_usage.hpp"
 #include "util.hpp"
 
+#include "jni_util/jni_utils.hpp"
+
 using std::string;
+using namespace realm::jni_util;
```

Comment:
delete "using" line - you already qualify the namespace in l 45 - which is better anyway.